docs/dev/README.md

```diff
@@ -1,36 +1,101 @@
-### Git
+# Building and Running the Code
 
-TripleA repositories may make use of [Git Large File Storage](https://git-lfs.github.com/) (LFS).  In order to work with such a repository, devs will need to have a Git LFS client installed.  Installation instructions for various platforms is available [here](https://github.com/git-lfs/git-lfs#getting-started).
+```
+./gradlew test
```

Comment:
`./gradlew check`

Reply:
that's an important command, the original docs had 'test' and 'check' command listed in other places. I updated the branch to have the check command included. It's a small decision to remove 'test' completely, that is not in scope of these updates as we are first consolidating, and then incrementally removing the out of date.

Reply:
4b2de734ac58af2caefb224eada04ed2e6b7e68f